src/app/detail/ui/TaskEditor.tsx

```diff
@@ -67,6 +68,12 @@ export const TaskEditor = ({
     }
   }
 
+  const _taskUpdateDebounced = async (title: string, details: string) => {
+    updateTaskDetail(title, details)
+  }
```

Comment:
Shouldn't `updateTaskDetail` be awaited here?

Reply:
I dont think this is necessary. the function is passed from server component and is awaited there.